src/ape/api/transactions.py

```diff
@@ -402,6 +413,14 @@ def show_gas_report(self, file: IO[str] = sys.stdout):
         Display a gas report for the calls made in this transaction.
         """
 
+    @raises_not_implemented
+    def show_traceback(self):
```

Comment:
I think `show_trace` is a little to close to this name, maybe `show_error_traceback` (and corresponding `error_traceback`)

Reply:
I agree. However, I was thinking - there is no reason to not have this option for non-failing receipts.. Like it still works, you just make it all the way to the `RETURN` opcode in the traceback. We can make it not be red or something.